.travis.yml

```diff
@@ -15,17 +15,19 @@ cache:
   - $HOME/.gradle/wrapper
 
 before_install:
-- echo "[INFO] BUILD_COMMIT> $BUILD_COMMIT"
+  - cat gradle.properties
+  - echo "[INFO] BUILD_COMMIT> $BUILD_COMMIT"
+  - sed -i -r  "/version/s/([[:digit:]]+\.[[:digit:]]+\.[[:digit:]]+)/\1-$BUILD_COMMIT/g" gradle.properties
```

Comment:
What's the idea?

Reply:
instead of having 5.2.4-SNAPSHOT --> 5.2.4-b266_g37d78f9d-SNAPSHOT

Reply:
This is at Travis only. Do you intend to release snapshot versions?
I just thought that release versions should not include -SNAPSHOT suffix.

Reply:
I do this already since a couple of months? I release them as [Pre-Release](https://github.com/vlsi/ksar/releases)
The regex just replace 5.2.4 with something .. so it can handle 5.2.4 or 5.2.4-SNAPSHOT

Reply:
I see

Reply:
any objections ?

Reply:
no, that's fine with me.
I think we might want to move that to `build.gradle`, however travis.yml is fine for now